component/README.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: pattern  
+title: Component  
+folder: component  
+permalink: /patterns/component/  
+categories: Behavioral  
+tags:  
+ - Java  

```

Comment:
I would suggest `Game programming` and `Decoupling` tags. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags